CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ find_package(Protobuf REQUIRED)
 include_directories(${PROTOBUF_INCLUDE_DIRS})
 
 add_subdirectory(third_party)
+include_directories(.)
```

Comment:
`include_directories(.)` seems too ambitious, may be add 'src' folder under root director and move `substrait` folder inside `src`, then replace `include_directories(.)` with `include_directories(src)`?

Anyway let's plan in another separate PR.